src/components/Trade/TradeForm/TradeForm.tsx

```diff
@@ -831,6 +833,28 @@ export const TradeForm = ({
                   if (!allowedSlippage) return false;
                   return slippage?.lt(allowedSlippage);
                 },
+                notEnoughFeeBalance: () => {
+                  const assetIn = getValues('assetIn');
+                  const assetInAmount = getValues('assetInAmount');
+
+                  let nativeAssetBalance = find(activeAccount?.balances, {
+                    assetId: '0'
+                  })?.balance;
+
+                  let balanceForFee = nativeAssetBalance;
+
+                  if (assetIn === '0' && assetInAmount && nativeAssetBalance) {
```

Comment:
NativeAssetBalance = feepaymentbalance?

Reply:
Naming is derived from the combined `balances` resolver, which resolves a `nativeAssetBalance` internally for `assetId = 0`